nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_Init.c

```diff
@@ -28,6 +28,7 @@ static NFCSTATUS phNciNfc_RegAllNtfs(void* pContext);
 /*Global Varibales for Init Sequence Handler*/
 phNciNfc_SequenceP_t gphNciNfc_InitSequence[] = {
     {&phNciNfc_InitReset, &phNciNfc_ProcessResetRsp},
+    /* In NCI1.x phNciNfc_DelayForResetNtf is not needed as we don't expect to receive any CoreResetNtf */
```

Comment:
I have reworked the comment following your suggestion.
While at it, looking into the 2 next sequences, gphNciNfc_ReleaseSequence and gphNciNfc_NfccResetSequence need to follow the same scheme for NCI2.0.

I missed it for RS2 and integrated a workaround in the client that hide it until now.
The update is available in the same branch.